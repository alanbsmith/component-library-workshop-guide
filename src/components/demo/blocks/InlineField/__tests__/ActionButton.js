import { generateSnapshots } from "__tests__/helpers";

import ActionButton from '../ActionButton';

generateSnapshots(ActionButton);
