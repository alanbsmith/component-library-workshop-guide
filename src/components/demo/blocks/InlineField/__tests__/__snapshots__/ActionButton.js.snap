// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton renders correctly 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.125rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  outline: none;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  width: 1.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.2);
  -ms-transform: scale(1.2);
  transform: scale(1.2);
}

.c0:hover span,
.c0:focus span {
  color: #9aa0ac;
}

<button
  className="c0"
/>
`;

exports[`IconButton should apply hoverDanger modifier 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.125rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  outline: none;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  width: 1.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.2);
  -ms-transform: scale(1.2);
  transform: scale(1.2);
}

.c0:hover span,
.c0:focus span {
  color: #9aa0ac;
}

.c0:hover span,
.c0:focus span {
  color: #DD3800;
}

<button
  className="c0"
/>
`;

exports[`IconButton should apply hoverInfo modifier 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.125rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  outline: none;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  width: 1.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.2);
  -ms-transform: scale(1.2);
  transform: scale(1.2);
}

.c0:hover span,
.c0:focus span {
  color: #9aa0ac;
}

.c0:hover span,
.c0:focus span {
  color: #0975E0;
}

<button
  className="c0"
/>
`;

exports[`IconButton should apply hoverSuccess modifier 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.125rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  outline: none;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  width: 1.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.2);
  -ms-transform: scale(1.2);
  transform: scale(1.2);
}

.c0:hover span,
.c0:focus span {
  color: #9aa0ac;
}

.c0:hover span,
.c0:focus span {
  color: #008900;
}

<button
  className="c0"
/>
`;

exports[`IconButton should apply hoverWarning modifier 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.125rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  outline: none;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  width: 1.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c0:hover,
.c0:focus {
  -webkit-transform: scale(1.2);
  -ms-transform: scale(1.2);
  transform: scale(1.2);
}

.c0:hover span,
.c0:focus span {
  color: #9aa0ac;
}

.c0:hover span,
.c0:focus span {
  color: #967100;
}

<button
  className="c0"
/>
`;
